packages/devtools_app/lib/main.dart

```diff
@@ -5,7 +5,7 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
-import 'src/analytics/stub_provider.dart'
+import 'src/analytics/provider.dart'
```

Comment:
can this be cleaned up to match our other conditional imports with
src/analytics/provider.dart internally using conditional imports so that call sites like this don't have to remember to add the conditional imports. Would be easy to accidentally only import 'src/analytics/provider.dart' and end up with stubbed out analytics on html.

Reply:
done - here and with the analytics.dart conditional files as well